translations/nl.po

```diff
@@ -460,14 +461,14 @@ msgstr "Contactgegevens"
 msgid "Park offer"
 msgstr "Aanbod en mogelijkheden"
 
-#: themes/contrib/gent_base/templates/custom/vesta/vesta--teaser.html.twig:29
-msgid "@name - openinghours and addresses"
-msgstr "@name - Openingsuren en adressen"
-
 #: themes/contrib/gent_base/templates/custom/vesta/vesta--teaser.html.twig:43
 msgid "More info and opening hours"
 msgstr "Meer info en openingsuren"
 
+#: themes/contrib/gent_base/templates/custom/vesta/vesta.html.twig:29
+msgid "@name - openinghours and addresses"
+msgstr "@name - Openingsuren en adressen"
```

Comment:
Met hoofdletter in beide talen of zonder? :)

Reply:
Geen idee maar deze PR verandert enkel de volgorde van deze strings, niet de vertaling zelf.